### ISSUE TITLE
Possible to compile asynchttpserver_parallel.nim?

### ISSUE THREAD

**Issue by devinus** [def-/nim-http-speedup#1]
Do I have to use your forked version of Nim to compile?

**Comment by def-** [comment]
Yeah, but that code is very experimental and doesn't even work correctly. I've heard that Araq is working on making this work in a proper way.

**Comment by simonkrauter** [comment]
I'm interested in such a solution too :)
